node/src/Pos/Communication/Tx.hs

```diff
@@ -60,27 +61,29 @@ submitAndSave enqueue txAux@TxAux {..} = do
 -- | Construct Tx using multiple secret keys and given list of desired outputs.
 prepareMTx
     :: TxMode ssc m
-    => (Address -> SafeSigner)
+    => [PendingTx]
+    -> (Address -> SafeSigner)
     -> NonEmpty Address
     -> NonEmpty TxOutAux
     -> AddrData m
     -> m (TxAux, NonEmpty TxOut)
-prepareMTx hdwSigners addrs outputs addrData = do
+prepareMTx pendingTxs hdwSigners addrs outputs addrData = do
     utxo <- getOwnUtxos (toList addrs)
-    eitherToThrow =<< createMTx utxo hdwSigners outputs addrData
+    eitherToThrow =<< createMTx pendingTxs utxo hdwSigners outputs addrData
 
 -- | Construct Tx using secret key and given list of desired outputs
 submitTx
     :: TxMode ssc m
-    => EnqueueMsg m
+    => [PendingTx]
+    -> EnqueueMsg m
     -> SafeSigner
     -> NonEmpty TxOutAux
     -> AddrData m
     -> m (TxAux, NonEmpty TxOut)
-submitTx enqueue ss outputs addrData = do
+submitTx pendingTxs enqueue ss outputs addrData = do
     let ourPk = safeToPublic ss
     utxo <- getOwnUtxoForPk ourPk
-    txWSpendings <- eitherToThrow =<< createTx utxo ss outputs addrData
+    txWSpendings <- eitherToThrow =<< createTx pendingTxs utxo ss outputs addrData
```

Comment:
Same deal, I think this is GH being conservative, we definitely ported the `OptimiseForHighThroughput` policy.